tests/integration/test_interaction.py

```diff
@@ -487,14 +482,17 @@ class AssessmentTestMixin(object):
     Provides helper methods for assessment tests
     """
     @staticmethod
-    def _wait_until_enabled(element):
+    def _wait_until_enabled(element, message=None):
         wait = WebDriverWait(element, 2)
-        wait.until(lambda e: e.is_displayed() and e.get_attribute('disabled') is None)
+        wait.until(lambda e: e.is_displayed() and e.get_attribute('disabled') is None, message)
+
+    def _just_wait(self, seconds):
```

Comment:
I don't think this is doing what you expect. And instead of waiting for a specified amount of time, it is much better to wait for an element to be in its expected state.

http://www.seleniumhq.org/docs/04_webdriver_advanced.jsp
`An implicit wait is to tell WebDriver to poll the DOM for a certain amount of time when trying to find an element or elements if they are not immediately available. The default setting is 0. Once set, the implicit wait is set for the life of the WebDriver object instance.`